hms-app/src/pages/Archive.tsx

```diff
@@ -1,18 +1,74 @@
-import React from 'react'
-import ideaData from '../test/TestIdeaData'
-import IdeaCardList from '../components/IdeaCardList'
-import { Idea } from '../common/types'
+import React, { useEffect, useState } from 'react'
+import { getListOfHackathons } from '../actions/GetBackendData'
+import HackathonDetails from '../components/HackathonDetails'
+import { Select } from '@mantine/core'
+
+interface HackathonPreview {
+  id: string
+  title: string
+}
 
 export default function Archive() {
+  const [selectedHackweek, setSelectedHackweek] = useState('')
+  const [hackathonList, setHackathonList] = useState({
+    errorhackathonList: false,
+    isLoadinghackathonList: true,
```

Comment:
Would also need to fix this everywhere
```suggestion
    errorHackathonList: false,
    isLoadingHackathonList: true,
```